.github/workflows/update_gradle_cache.yml

```diff
@@ -10,7 +10,7 @@ on:
       - "gradle**" # covers gradle folder, gradle.properties, gradlew
       - "build.gradle*"
       - "settings.gradle*"
-      - "src/main/resources/META-INF/*_at.cfg" # access transformers
+      - "src/main/resources/*_at.cfg" # access transformers
```

Comment:
Is this just a leftover from FG5 -> RFG? Or is there some other reason for this

Reply:
We had previously changed the location of this file to fix an issue with the RFG port. The action just was not updated to it when the change originally happened. Since this PR is already touching actions scripts (therefore requiring a cache rebuild), I thought it would make the most sense to fix this at the same time.